### ISSUE TITLE
Update Eclipse Paho website with new repo links

### ISSUE THREAD

**Issue by kamilfb** [eclipse/paho.mqtt-spy#8]
Update Eclipse Paho website with:

- new Eclipse repo links
- short description of mqtt-spy

**Comment by icraggs** [comment]
Hi Kamil.  We should have a new page for mqtt-spy, with similar content to the other components.  Do you need help with this? (I'd like it to be done before the upcoming Paho 1.3 release)

**Comment by kamilfb** [comment]
@icraggs, yes - I need some help with that. I'm more than happy to provide the content, but not sure how to do it.
